x-pack/plugins/ml/public/application/components/import_export_jobs/import_jobs_flyout/cannot_import_jobs_callout.tsx

```diff
@@ -64,14 +64,23 @@ const SkippedJobList: FC<{ jobs: SkippedJobs[] }> = ({ jobs }) => (
   <>
     {jobs.length > 0 && (
       <>
-        {jobs.map(({ jobId, missingIndices }) => (
+        {jobs.map(({ jobId, missingIndices, missingFilters }) => (
           <EuiText size="s">
             <h5>{jobId}</h5>
-            <FormattedMessage
-              id="xpack.ml.importExport.importFlyout.cannotImportJobCallout.missingIndex"
-              defaultMessage="Missing index {num, plural, one {pattern} other {patterns}}: {indices}"
-              values={{ num: missingIndices.length, indices: missingIndices.join(',') }}
-            />
+            {missingIndices.length > 0 && (
+              <FormattedMessage
+                id="xpack.ml.importExport.importFlyout.cannotImportJobCallout.missingIndex"
+                defaultMessage="Missing index {num, plural, one {pattern} other {patterns}}: {indices}"
+                values={{ num: missingIndices.length, indices: missingIndices.join(',') }}
+              />
+            )}
+            {missingFilters.length > 0 && (
+              <FormattedMessage
+                id="xpack.ml.importExport.importFlyout.cannotImportJobCallout.missingFilters"
+                defaultMessage="Missing {num, plural, one {filter} other {filters}}: {filters}"
```

Comment:
```suggestion
                defaultMessage="Missing {num, plural, one {filter list} other {filter lists}}: {filters}"
```

Reply:
Updated in https://github.com/elastic/kibana/pull/107416/commits/e64c591042b919577dc0e9c81c1ed2e92b043579